src/vacancies/vacancies-service.js

```diff
@@ -96,6 +96,23 @@ class VacancyService extends Service {
       return 'must be less than 30 characters';
     }
   }
+
+  validateTags(tags) {
+    if (tags.length > 10) {
+      return 'must be no more than 10 skills';
+    }
+  }
+
+  validateTagLength(tags) {
+    tags = tags.toString();
+    if (tags.length < 3) {
+      return 'must be at least 3 characters';
+    }
+
+    if (tags.length > 30) {
+      return 'must be no more than 30 characters';
+    }
+  }
```

Comment:
Although we call this many times, we accept only one tag at a time in this funciton. So let's have the signature reflect this for readability. 

```suggestion
  validateTags(tags) {
    if (tags.length > 10) {
      return 'must be no more than 10 skills';
    }
  }

  validateTagLength(tag) {
    tag = tag.toString();
    if (tag.length < 3) {
      return 'must be at least 3 characters';
    }

    if (tag.length > 30) {
      return 'must be no more than 30 characters';
    }
  }
```